src/utils/AppResponseConverter.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Response Converter
+ * Define utils to convert data from PagoPaAPI interfaces to controller interfaces and viceversa
+ */
+
+import { Either, Left, Right } from "fp-ts/lib/Either";
+import { AckResult } from "../api/types/BaseResponse";
+import { NotificationSubscriptionResponse } from "../api/types/NotificationSubscriptionResponse";
+import { ControllerError } from "../enums/ControllerError";
+import { NotificationSubscriptionResponseApp } from "../types/NotificationSubscriptionResponseApp";
+
+// Data converter for controllers to translate API responses to Controller responses
+export class AppResponseConverter {
+  public static getNotificationSubscriptionResponseFromAPIResponse(
```

Comment:
`isNotificationSubscriptionSuccess` ? (use is* for boolean methods)

Reply:
It's not a boolean method. It just define a response for a specific request